packages/p2p-media-loader-hlsjs/src/services/fragment-loader.ts

```diff
@@ -40,11 +51,13 @@ export class FragmentLoaderBase implements Loader<FragmentLoaderContext> {
 
     const playlist = this.identifyPlaylist(context);
     if (!playlist) {
-      return this.defaultLoader.load(context, config, callbacks);
+      this.defaultLoader = this.createDefaultLoader();
+      this.defaultLoader.load(context, config, callbacks);
```

Comment:
I think this is required:
```suggestion
      this.defaultLoader = this.createDefaultLoader();
      this.stats = this.defaulltLoader.stats;
      this.defaultLoader.load(context, config, callbacks);
```